pkg/pipelineascode/pipelineascode.go

```diff
@@ -64,6 +65,15 @@ func (p *PacRun) Run(ctx context.Context) error {
 		p.manager.Enable()
 	}
 
+	// set params for the console driver, only used for the custom console ones
+	cp := customparams.NewCustomParams(p.event, repo, p.run, p.k8int, p.eventEmitter)
```

Comment:
I see this changes only for custom console 

So do we need to check if custom console configurations are set or not

If not set then we should skip this entire part?

Reply:
but yeah we don't know which driver is set from the interface we could add this but that's fine to have it to setparams by default, it can be used in the future and maybe not worse the refacotring